### PR TITLE
Closes AgileVentures/MetPlus_tracker#311

### PR DESCRIPTION
**Pull request by patmbolger**

### CODE REVIEW

File: app/views/agency_people/_form.html.haml [hunk 1]
```diff
@@ -1,84 +1,86 @@
-= form_for agency_person, url: url, method: method,            |
-          html: {class: 'form-horizontal'} do |person_form|    |
+= form_for agency_person, url: url, method: method do |person_form|
+  -# html: {class: 'form-horizontal'} do |person_form|
```
Comment (joaopapereira) [inline app/views/agency_people/_form.html.haml]: If we are no longer using we should remove

File: app/views/agency_people/_form.html.haml [hunk 2]
```diff
@@ -1,84 +1,86 @@
...
 
   -# branch assignment (belongs_to)
   .form-group
-    = person_form.label :branch_id, 'Branch', class: 'control-label col-sm-2'
-    .col-sm-4
-      = person_form.collection_select(:branch_id, Branch.all, :id, :code, |
-            {include_blank: true}, class: 'form-control')     |
+    = person_form.label :branch_id, 'Branch', class: 'col-sm-2'
+    -# .col-sm-4
```
Comment (joaopapereira) [inline app/views/agency_people/_form.html.haml]: If we are no longer using we should remove

File: app/views/agency_people/_form.html.haml [hunk 3]
```diff
@@ -1,84 +1,86 @@
...
-          -# However, the controller does not receive the role id in the list of
-          -# role id's sent in the params, and thus it appears that the
-          -# agency admin checkbox has been unset.  This causes a model
-          -# validation error when no such error has been caused by the user.
-          -# This fix is described here:
-          -# http://stackoverflow.com/questions/4727974/
-          -# how-to-post-submit-an-input-checkbox-that-is-disabled
-          -# (see the second solution listed)
+    = person_form.label :agency_roles, 'Roles', class: 'col-sm-2'
+    -# .col-sm-10
```
Comment (joaopapereira) [inline app/views/agency_people/_form.html.haml]: If we are no longer using we should remove

**Comment by joaopapereira** [comment]
Looks good, please review my comments

**Comment by patmbolger** [comment]
Removed comments.

**Comment by joaopapereira** [comment]
Looks good going to merge

**Pull request merged**